packages/jqr/install

```diff
@@ -2,8 +2,15 @@
 set -x
 set -e
 
+OS_DISTRIBUTION=$(lsb_release -cs)
+
 apt-get update -qq
 apt-get install -y software-properties-common python-software-properties
-add-apt-repository -y ppa:opencpu/jq
-apt-get update -qq
+
+# these sysdeps are available by default in bionic
+if [ ${OS_DISTRIBUTION} != "bionic" ]; then
+  add-apt-repository -y ppa:opencpu/jq
```

Comment:
Does not support bionic: https://launchpad.net/~opencpu/+archive/ubuntu/jq

Reply:
Didn't comment on this one. :)

Reply:
Yes 😉 Put it here for my own sanity

Reply:
Probably should switch this to check if the distribution is xenial, since whatever comes after bionic will likely have `jq` as well.